packages/flutter_tools/test/commands.shard/hermetic/build_fuchsia_test.dart

```diff
@@ -16,104 +17,106 @@ import 'package:flutter_tools/src/project.dart';
 import 'package:meta/meta.dart';
 import 'package:mockito/mockito.dart';
 import 'package:process/process.dart';
-import 'package:flutter_tools/src/globals.dart' as globals;
 
 import '../../src/common.dart';
 import '../../src/context.dart';
 import '../../src/mocks.dart';
 
+// Defined globally for mocks to use.
+FileSystem fileSystem;
```

Comment:
Should it be `kFileSystem`?

Reply:
but Its not a konstant

Reply:
I thought there was some naming convention for globals that conflated constants and globals but I just misread.  It's all so Hungarian...